cypress/e2e/crypto/crypto.spec.ts

```diff
@@ -60,7 +61,6 @@ const testMessages = function(this: CryptoTestContext) {
     // check the invite message
     cy.contains(".mx_EventTile_body", "Hey!").closest(".mx_EventTile").within(() => {
         cy.get(".mx_EventTile_e2eIcon_warning").should("not.exist");
-        cy.get(".mx_EventTile_receiptSent").should("exist");
```

Comment:
At this point in the test this message is not the latest event in the room, so no longer shows the sent receipt